README.md

```diff
@@ -162,7 +162,15 @@ npm uninstall simple-git-hooks
 
 ### I want to skip git hooks!
 
-You should use `--no-verify` option
+If you need to bypass install hooks at all, for example on CI, you can use `SKIP_INSTALL_SIMPLE_GIT_HOOKS` environment variable at the first place.
```

Comment:
Do we need to handle delete simple git hooks in some way? Like, what will happen when we have `SKIP_INSTALL_SIMPLE_GIT_HOOKS=1` and we try to delete hooks?

--

wyt?

Reply:
I don't quite to follow, `SKIP_INSTALL_SIMPLE_GIT_HOOKS=1` means no hooks to be installed at all, what to delete then?